### PR TITLE
[docs] Documentation changes from APM Server

### PR DESCRIPTION
**Pull request by bmorelli25**
This PR persists some of the documentation changes needed in https://github.com/elastic/apm-server/pull/2542. This PR also includes a commit (b2b9d36) with Asciidoctor fixes.

### CODE REVIEW

File: libbeat/docs/monitoring/monitoring-metricbeat.asciidoc [hunk 1]
```diff
@@ -108,7 +108,6 @@ locations.
 endif::[]
 
 . Enable the `beat-xpack` module in {metricbeat}. +
-endif::[]
```
Comment (dedemorton) [inline libbeat/docs/monitoring/monitoring-metricbeat.asciidoc]: thanks for catching this!

File: libbeat/docs/monitoring/monitoring-beats.asciidoc [hunk 2]
```diff
@@ -6,7 +6,12 @@
 --
 
 You can use the {stack} {monitor-features} to gain insight into the health of
+ifndef::apm-server[]
 {beatname_uc} agents running in your environment.
```
Comment (dedemorton) [inline libbeat/docs/monitoring/monitoring-beats.asciidoc]:
```suggestion
{beatname_uc} instances running in your environment. 
```

Comment (dedemorton) [reply libbeat/docs/monitoring/monitoring-beats.asciidoc]: Changed this to be consistent with a naming discussion we had in a different PR.

File: libbeat/docs/monitoring/monitoring-metricbeat.asciidoc (hunk 1 above)
Comment (bmorelli25) [reply libbeat/docs/monitoring/monitoring-metricbeat.asciidoc]: Asciidoctor is the real hero here!

**Pull request merged**